dev-python/pythondialog/pythondialog-3.5.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python module for making simple text/console-mode user interfaces"
+HOMEPAGE="http://pythondialog.sourceforge.net/"
+SRC_URI="mirror://sourceforge/pythondialog//${PV}/python3-${P}.tar.bz2"
```

Comment:
why the double `//`

Reply:
ohh, I kept trying to figure out what you meant.  I'm not sure; I simply reused those vars from the previous versions of the ebuild.